src/arch/src/aarch64/gic.rs

```diff
@@ -19,7 +19,7 @@ pub enum Error {
 type Result<T> = result::Result<T, Error>;
 
 /// Trait for GIC devices.
-pub trait GICDevice {
+pub trait GICDevice: Send {
```

Comment:
It looks like the `Send` bound is only required because `EventHandler` has a `Send` bound as well (in `polly/src/event_manager.rs:66`).  I think we can remove that, and by doing so we no longer have to add this one, nor the `unsafe impl`s below.

Reply:
Good catch! Since the Handler is wrapped in a Mutex it doesn't have to be Send itself.
I will remove the `Send` bound from `EventHandler` and that should remove the need to mark `GICDevice` as `Send` too.

Reply:
This is done in the latest PR version.